iroha_p2p/src/peer.rs

```diff
@@ -468,24 +600,24 @@ pub async fn send_message(stream: &mut OwnedWriteHalf, data: &[u8]) -> Result<()
     Ok(())
 }
 
-/// Peer's identification.
-#[derive(Clone, Debug, PartialEq, Eq, PartialOrd, Ord, Hash, Encode, Decode)]
-pub struct PeerId {
-    /// Address of the Peer's entrypoint.
-    pub address: String,
-    /// Public Key of the Peer.
-    pub public_key: Option<iroha_crypto::PublicKey>,
-}
-
-impl From<iroha_data_model::peer::Id> for PeerId {
-    fn from(id: Id) -> Self {
-        Self {
-            address: id.address,
-            public_key: Some(id.public_key),
+/// Produces pseudorandom connection id from address
+fn get_connection_id(addr: io::Result<SocketAddr>) -> ConnectionId {
```

Comment:
Isn't it better then to panic outside of the function and just accept `addr: SocketAddr` as a param?

Reply:
I wanted to simplify the code there, by pulling the match here.

Reply:
Now that we want to simply panic, you don't even need a match you can just `expect`

Reply:
It is just strange to pass a result to the function which is guaranteed to panic if the result is Err.

Reply:
You've got a point, will try to simplify...

Reply:
Fixed.